metal/tuners/random_tuner.py

```diff
@@ -66,7 +65,22 @@ def search(
             # Unless seeds are given explicitly, give each config a unique one
             if config.get("seed", None) is None:
                 config["seed"] = self.seed + i
-            model = self.model_class(*init_args, **init_kwargs, **config)
+
+            # Integrating generated config into init kwargs and train kwargs
+            init_kwargs = recursive_merge_dicts(init_kwargs, config)
+            train_kwargs = recursive_merge_dicts(train_kwargs, config)
+
+            # Removing potential duplicate keys -- error occurs without this!
```

Comment:
??

Reply:
To make sure we don't have duplicate keys when we're doing model search -- if init_kwargs and config have duplicate keys, we error out.